acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/keymanager/ImportKeystoresAcceptanceTest.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.tests.keymanager;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.Matchers.hasItem;
+
+import tech.pegasys.web3signer.core.service.http.handlers.keymanager.imports.ImportKeystoresRequestBody;
+import tech.pegasys.web3signer.core.signing.KeyType;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.apache.tuweni.bytes.Bytes;
+import org.jdbi.v3.core.Jdbi;
+import org.junit.jupiter.api.Test;
+
+public class ImportKeystoresAcceptanceTest extends KeyManagerTestBase {
+
+  private static final String PUBLIC_KEY =
+      "0x98d083489b3b06b8740da2dfec5cc3c01b2086363fe023a9d7dc1f907633b1ff11f7b99b19e0533e969862270061d884";
+
+  @Test
+  public void invalidRequestBodyReturnsError() {
+    setupSignerWithKeyManagerApi();
+    final Response response = callImportKeystores("{\"invalid\": \"json body\"}");
+    response.then().assertThat().statusCode(400);
+  }
+
+  @Test
+  public void validRequestBodyReturnsSuccess() throws IOException, URISyntaxException {
+    setupSignerWithKeyManagerApi();
+    final Response response = callImportKeystores(composeRequestBody());
+    response
+        .then()
+        .contentType(ContentType.JSON)
+        .assertThat()
+        .statusCode(200)
+        .body("data.status", hasItem("IMPORTED"));
+  }
+
```

Comment:
Be good to have test that shows when importing multiple keys that the invalid isn't imported but the valid ones are